### ISSUE TITLE
Android versionCheck.needUpdate() returns undefined

### ISSUE THREAD

**Issue by MichaelAmadheo** [kimxogus/react-native-version-check#177]
This is working on iOS but not working for android, it returns undefined

**Comment by lundjrl** [comment]
@MichaelAmadheo Same here but our app is currently in development and does not have an app store presence on either platform. Is this the same for you?

**Comment by lapwil** [comment]
yep same for me, but only on development env. everything works on staging and release.

**Comment by MichaelAmadheo** [comment]
@lundjrl we have the app presence both on Playstore and Appstore. It used to return all the information just like iOS, but just recently it returns undefined and we cannot get the url for playstore

**Comment by adelchms96** [comment]
@MichaelAmadheo i solved this by using   `VersionCheck.getStoreUrl()` for android and  `versionCheck.needUpdate()` for ios for now !

**Comment by NageshA** [comment]
@MichaelAmadheo use the below code. In Android I used was able to solve it by this code `latestVersion` we need to pass to .needUpdate function.

```
const res = await fetch(
        `https://play.google.com/store/apps/details?id=${VersionCheck.getPackageName()}&hl=en`,
      );
      const text = await res.text();
      let latestVersion;
      const match = text.match(/\[\[\["([\d.]+?)"\]\]/);
      if (match) {
        latestVersion = match[1].trim();
      }
      
      let updateNeeded = await VersionCheck.needUpdate({
        latestVersion,
      });
```

**Comment by ngotra2710** [comment]
I am using the latest version v3.4.7 but the function getLatestVersion is not working.
I figured out that the returned HTML from google play does not match the regex.
text.match(/Current Version.+?>([\d.-]+)<\/span>/); and text.match(/\[\[\["([\d.]+?)"\]\]/) return null.
I search entire of the return HTML and could not find the version string that I expect.

I see that if I access this URL https://play.google.com/store/apps/details?id=${opt.packageName}&hl=en and want to see the version number, I have to click on the "About this app" right arrow. If I don't click it, the version string does not exist in the HTML.
